### PR TITLE
Card update

### PR DESCRIPTION
**Pull request by Duncid**
## Description

Bringing consistency to different "Card" components:
- `Card` > `ValueCard`
- `CardButton` > `Card`
- `ValueCard` makes use of `Card`
- `AssistantPreview` > To be replaced by `AssistantCard`
- `Citation` don't change (already based on `Card`

Creating a new CardGrid component to simply create a responsive Card grid.

## Risk
Needs to be careful at Deploy as components will change names.

## Deploy Plan
Merge, deploy a new Sparkle, new PR for making use of the new Sparkle.

### CODE REVIEW

File: sparkle/src/components/AssistantCard.tsx [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+  }
+>(({ className, ...props }, ref) => {
+  return (
+    <Button
+      ref={ref}
+      size="mini"
+      variant="outline"
+      icon={MoreIcon}
+      className={className}
+      {...props}
```
Comment (JulesBelveze) [inline sparkle/src/components/AssistantCard.tsx:84]: I don't think you wanna spread the props here

**Review by JulesBelveze — COMMENTED** [review]
Left a few comments. I'm not sure if the `ValueCard` component still adds a lot of value, could we derive it from the `Card` itself?

**Comment by Duncid** [comment]
Thanks @JulesBelveze !
Made changes accordingly.

**Review by JulesBelveze — APPROVED** [review]
Looks 🔥 thx 🙏🏼

**Pull request merged**